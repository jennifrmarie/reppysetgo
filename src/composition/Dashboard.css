.date-section {
    width: 55%;
    max-width: 400px;
    min-width: 400px;
    display: flex;
    flex-direction: column;
    background-repeat: no-repeat;
    align-items: center; 
    justify-content: center;;
}

.calendar_container {
    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: #ece8e8fb;
    min-height: 350px;
    /* background-image: url('https://images.pexels.com/photos/669576/pexels-photo-669576.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260'); */
}

.li-date {
    font-size: 20px;
    letter-spacing: .3em;
}
.logo__dashboard {
    background-image: url('https://i.imgur.com/jBMcwAH.png');

    background-color: #ece8e8fb;
    background-position: center;
    font-size: 20px;
    position: relative;
    min-height: 200px;
    max-width: 100%;
    background-repeat: no-repeat;
}

.calendar {
    background-color: black;
    border: 2px solid white;
    opacity: 60%;
    padding: 10px;
    display: flex;
}

.calendar:hover {
    color: black;
}

.date-selector {
    /* position: absolute;
    top: 40%;
    left: 15%;
    right: 8%; */
    width: 300px;
    /* margin-left: 30px; */
}

.dashboard__background {
    min-height: 100vh;
    background-image: url('https://images.pexels.com/photos/669576/pexels-photo-669576.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260');
    background-size: cover;
    background-repeat: no-repeat;
    /* background-size: 100%; */
}

.DayPicker:not(.DayPicker--interactionDisabled) .DayPicker-Day:not(.DayPicker-Day--disabled):not(.DayPicker-Day--selected):not(.DayPicker-Day--outside):hover {
    color: black;
}


