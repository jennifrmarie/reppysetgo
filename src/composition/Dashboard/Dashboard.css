.date-section {
    width: 55%;
    margin: 0;
    max-width: 400px;
    min-width: 300px;
    display: flex;
    flex-direction: column;
    background-repeat: no-repeat;
    align-items: center; 
    justify-content: center;;
}

.calendar_container {

    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: #ece8e8fb;
    min-height: 350px;
    width: 100%;
    padding: 20px;
}

.li-date {
    align-self: center;
    display: flex;
    letter-spacing: .3em;
    color: white; 
    font-size: 17px;
    font-weight: 5px;
    padding: 20px;
    background-color: black;
    -webkit-box-shadow: 0px 4px 0px #36424b;
-moz-box-shadow: 0px 4px 0px #36424b;
box-shadow: 0px 4px 0px #36424b;
}
.li-date:hover {
    cursor: pointer;
    transform: scale(1.2) perspective(1px)
}
.logo__dashboard {
    background-image: url('https://i.imgur.com/jBMcwAH.png');
    background-size: 60%;
    background-color: #ece8e8fb;
    background-position: center;
    font-size: 20px;
    position: relative;
    min-height: 200px;
    max-width: 100%;
    padding: 20px;
    background-repeat: no-repeat;
}

.dashboard__list {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-content: center;
    width: 100%;
    min-width: 200px;
    padding: 10px;
}

.workout-name {
    color: black;
    align-self: center;
    display: flex;
    text-align: center;
    margin: auto;
}

.logout_button {
    background-color: #000000;
    color: white;
    font-size: 15px;
    padding: 10px;
}

.dashboard_delete:hover {
    cursor: pointer;
}

.dashboard_delete {
    margin: auto;
}

.edit_button {
    padding-right: 5px;
}

.workout-weight {
    color: black;
    padding-right:20px ;
    align-self: center;
    padding-right: 50px;
}

.fa-chevron-down > path:hover {
    transform: scale(1.5) perspective(1px)
}

.fa-edit > path {
    color: #000000;
    position: relative;
}

.fa-edit > path:hover {
    color: #ad4141fb;
    background-color: black;
}

.fa-trash-alt > path {
    color: #000000;
    position: relative;
}

.fa-trash-alt > path:hover {
    color: #ad4141fb;
}

.nav_buttons {
    display: flex;
    justify-self: flex-end;
    margin-left: auto;
}
.sets_reps {
    align-self: center;
    color: black;
}

.dashboard__workout {
    color: black;
    width: 40%;
    
    padding: 10px;
    margin: 5px;
    background-color: #e6e6e6fb;
    letter-spacing: .2em;
    align-self: center;
    justify-content: center;
    display: flex;
    flex-direction: row;
    line-height: 1.5em;
    font-size: 20px;
}

.dashboard__date {
    color: black;
    padding-right: 30px;
    align-self: center;
}

.calendar {
    background-color: black;
    border: 2px solid white;
    opacity: 60%;
    padding: 10px;
    display: flex;
}

.calendar:hover {
    color: black;
}

.date-selector {
    width: 300px;
}

.dashboard__background {
    min-height: 100vh;
    width: 100%;
    background-image: url('https://images.pexels.com/photos/669576/pexels-photo-669576.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260');
    background-size: cover;
    background-repeat: no-repeat;
    display: flex;
    align-content: center;
    flex-direction: column;
    margin: 0;
}

.DayPicker:not(.DayPicker--interactionDisabled) .DayPicker-Day:not(.DayPicker-Day--disabled):not(.DayPicker-Day--selected):not(.DayPicker-Day--outside):hover {
    color: black;
}

@media only screen and (max-width: 900px) {
    .dashboard__list {
        display: flex;
        flex-direction:column;
        justify-content: center;
        align-items: center;
    }
    .dashboard__date {
        margin: auto;
        
    }

    .logo__dashboard {
        background-size: 100%;
    }
    .dashboard__workout {
        display: flex;
        font-size: 12px;
        flex-direction: row;
        flex-shrink: 1;
        width: 80%;
    }
    .nav_buttons {
        margin-right: auto;
    }
}

@media only screen and (max-width: 1190px) and (min-width: 900px) {
    .dashboard__list {
        display: flex;
        flex-direction: column;

    }

    .dashboard__workout {
        width: 60%;
    }
}


