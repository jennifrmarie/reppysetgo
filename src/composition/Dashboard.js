import React, { Component } from 'react'
import DayPicker from 'react-day-picker';
import 'react-day-picker/lib/style.css';
import moment from 'moment';
import AppContext from '../AppContext'
import './Dashboard.css'
import { withRouter } from 'react-router-dom'
import WorkoutForm from './WorkoutForm';

class Dashboard extends React.Component {
  constructor(props) {
    super(props);
    this.handleDayClick = this.handleDayClick.bind(this);
    this.state = {
      selectedDay: undefined,
    };
  }
static contextType = AppContext

  handleWeekClick = (day, days, e) => {
    this.context.setDays(days)
    this.context.setDay(day)
  };

  handleDayClick(day, modifiers = {}) {
    if (modifiers.disabled) {
      return;
    }
    this.setState({
      selectedDay: modifiers.selected ? undefined : day,
    });
  }

  // handleDateClicked = (index) => {
  // //  const index = e.currentTarget.getAttribute("data-index")
  //  this.context.handleDateClicked(index)
  // //  this.props.history.push("/add-workout")
  // }


render() {
    const { selectedDays } = this.context;
    
    const daysAreSelected = selectedDays.length > 0;
    const modifiers = {
      selectedRange: daysAreSelected && {
        from: selectedDays[0],
        to: selectedDays[0],
      },
      selectedRangeStart: daysAreSelected && selectedDays[0],
      selectedRangeEnd: daysAreSelected && selectedDays[0],
      daysHighlighted: selectedDays && this.state.selectedDay
    };
    const modifiersStyles = {
      daysHighlighted: {
        backgroundColor: 'white',
        color: 'black',
      },
      outside: {
        backgroundColor: 'black'
      }
    };
    let dates = this.context.selectedDays
    const date = this.state.selectedDay
    console.log(date)
    
    // const DateList = () => (
    //   <div>
    //   <ul className="date-selector">
    //     {dates.map((date, index) => 
    //     <li key={index} data-index={index} className="li-date" 
    //     onClick={() => this.props.history.push(`/add-workout/${date}`) && <WorkoutForm date={date} />}>
    //       {moment(date).format('MMM Do, YYYY')}
    //     </li>
    //     )}
    //   </ul>
    //   </div>
    // );

    const SingleDate = () => (
    <div className='li-date' onClick={() => this.props.history.push(`/add-workout/${date}`) && <WorkoutForm date={this.context.selectedDay} />}>
        {moment(date).format('MMM Do, YYYY')}
      </div>
    )



    return (
      <div className="date-section">
        <h1>REPPY, SET, GO!</h1>
        <h3>click on a day to get started</h3>
        <div className="calendar">
        <DayPicker
          showWeekDays
          className="Selectable"
          selectedDays={this.context.selectedDays}
          selectedDay={this.state.selectedDay}
          onDayClick={this.handleDayClick} 
          modifiers={modifiers}
          modifiersStyles={modifiersStyles}
          // showWeekNumbers
          onWeekClick={this.handleWeekClick}
        />
        </div>
        {/* {this.context.selectedDays} */}
        {/* {this.state.selectedDay ? this.state.selectedDay.toLocaleString() :  */}
        {/* <DateList />  */}
        <SingleDate />

        {/* <Link to="/add-workout"> Add Workout </Link> */}
        </div>
    );
  }
}

export default withRouter(Dashboard)