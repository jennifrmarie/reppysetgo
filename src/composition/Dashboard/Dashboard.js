import React, { Component } from 'react'
import { Link } from 'react-router-dom'
import DayPicker from 'react-day-picker';
import 'react-day-picker/lib/style.css';
import moment from 'moment';
import AppContext from '../../AppContext'
import './Dashboard.css'
import WorkoutForm from '../WorkoutForm/WorkoutForm';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { faChevronDown } from '@fortawesome/free-solid-svg-icons';

export default class Dashboard extends Component {
  constructor(props) {
    super(props);
    this.handleDayClick = this.handleDayClick.bind(this);
    this.state = {
      selectedDay: undefined,
      isEmptyState: true
    };
  }
static contextType = AppContext

componentDidMount() {
  this.context.getItems()
}

  handleWeekClick = (day, days, e) => {
    this.context.setDays(days)
    this.context.setDay(day)
  };

  handleDayClick(day, modifiers = {}) {
    if (modifiers.disabled) {
      return;
    }
    this.setState({
      selectedDay: modifiers.selected ? undefined : day,
    });
  }

  triggerWorkoutForm = () => {
    this.setState({
      ...this.state,
      isAddWorkoutState: true,
      isEmptyState: false
    })
  }

  handleLogout = () => {
    localStorage.clear()
    this.props.history.push('/')
  }


render() {
    const { selectedDays } = this.context;
    
    const daysAreSelected = selectedDays.length > 0;
    const modifiers = {
      selectedRange: daysAreSelected && {
        from: selectedDays[0],
        to: selectedDays[0],
      },
      selectedRangeStart: daysAreSelected && selectedDays[0],
      selectedRangeEnd: daysAreSelected && selectedDays[0],
      daysHighlighted: selectedDays && this.state.selectedDay
    };
    const modifiersStyles = {
      daysHighlighted: {
        backgroundColor: 'white',
        color: 'black',
      },
    };
    const date = this.state.selectedDay
    const dateid = moment(date).format('MMDDYYYY')

    const SingleDate = (index) => (
      <button key={index} 
        className='li-date' 
        onClick={() => this.props.history.push(`/add-workout/${dateid}`)}
      >
        {moment(date).format('MMM Do, YYYY')}
      </button>
    )

    let items = this.context.items

    return (
      <div className="dashboard__background">
        <div className="logo__dashboard">
        <nav><button className="logout_button" onClick={this.handleLogout}>Logout</button></nav>
        </div>
        
        <div className="calendar_container">
        
        <div className="date-section">
          
          <DayPicker
            showWeekDays
            className="Selectable"
            selectedDays={this.context.selectedDays}
            selectedDay={this.state.selectedDay}
            onDayClick={this.handleDayClick} 
            modifiers={modifiers}
            modifiersStyles={modifiersStyles}
          />
          <FontAwesomeIcon icon={faChevronDown}></FontAwesomeIcon>
        </div>
        <SingleDate />
        </div>
      
      <ul className="dashboard__list">
        {
          items.map((item, index) => 
            <li className="dashboard__workout" key={index}>
              <div className="dashboard__date">{moment(item.date).format('MM-DD-YY')}</div>
              <span className="workout-name">{item.name}{''}</span>
                <span className="sets_reps">{item.sets}{"X"}
                {item.reps}{''}{":"}{'   '}</span>
                <span className="workout-weight">{" "}{item.weight}{"  lbs "}</span>
                <span className="nav_buttons">
                <Link to={'/edit-workout/' + item.id} tag='button' className="edit_button"><FontAwesomeIcon icon="edit" color="black" /></Link>{'  '}
                <div className="dashboard_delete" onClick={e => this.context.removeItem(item.id)}><FontAwesomeIcon icon="trash-alt" /></div>
              </span>
            </li>
          )
        }
      </ul>
        {this.state.isAddWorkoutState && <WorkoutForm dateId={dateid}/>} 
    </div>
    );
  }
}