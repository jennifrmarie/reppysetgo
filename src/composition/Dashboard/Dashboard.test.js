import React from 'react';
import ReactDOM from 'react-dom';
import Dashboard from './Dashboard';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Dashboard />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Dashboard name="Messages" unread={4}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });