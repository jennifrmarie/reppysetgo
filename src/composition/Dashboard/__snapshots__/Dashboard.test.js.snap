// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="dashboard__background"
>
  <div
    className="logo__dashboard"
  >
    <nav>
      <button
        className="logout_button"
        onClick={[Function]}
      >
        Logout
      </button>
    </nav>
  </div>
  <div
    className="calendar_container"
  >
    <div
      className="date-section"
    >
      <div
        className="DayPicker Selectable"
        lang="en"
      >
        <div
          className="DayPicker-wrapper"
          onKeyDown={[Function]}
          tabIndex={0}
        >
          <div
            className="DayPicker-NavBar"
          >
            <span
              aria-label="Previous Month"
              className="DayPicker-NavButton DayPicker-NavButton--prev"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex="0"
            />
            <span
              aria-label="Next Month"
              className="DayPicker-NavButton DayPicker-NavButton--next"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex="0"
            />
          </div>
          <div
            className="DayPicker-Months"
          >
            <div
              className="DayPicker-Month"
              role="grid"
            >
              <div
                className="DayPicker-Caption"
                role="heading"
              >
                <div
                  onKeyUp={[Function]}
                >
                  March 2020
                </div>
              </div>
              <div
                className="DayPicker-Weekdays"
                role="rowgroup"
              >
                <div
                  className="DayPicker-WeekdaysRow"
                  role="row"
                >
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Sunday"
                    >
                      Su
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Monday"
                    >
                      Mo
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Tuesday"
                    >
                      Tu
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Wednesday"
                    >
                      We
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Thursday"
                    >
                      Th
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Friday"
                    >
                      Fr
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Saturday"
                    >
                      Sa
                    </abbr>
                  </div>
                </div>
              </div>
              <div
                className="DayPicker-Body"
                role="rowgroup"
              >
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Mar 01 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={0}
                  >
                    1
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Mar 02 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    2
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Mar 03 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    3
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Mar 04 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    4
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Mar 05 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    5
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Mar 06 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    6
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Mar 07 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    7
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Mar 08 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    8
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Mar 09 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    9
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Mar 10 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    10
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Mar 11 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    11
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Mar 12 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    12
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Mar 13 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    13
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Mar 14 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    14
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Mar 15 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    15
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Mar 16 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    16
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Mar 17 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    17
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Mar 18 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    18
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Mar 19 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    19
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Mar 20 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    20
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Mar 21 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    21
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Mar 22 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    22
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Mar 23 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    23
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Mar 24 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    24
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Mar 25 2020"
                    aria-selected={false}
                    className="DayPicker-Day DayPicker-Day--today"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    25
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Mar 26 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    26
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Mar 27 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    27
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Mar 28 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    28
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Mar 29 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    29
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Mar 30 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    30
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Mar 31 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    31
                  </div>
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-chevron-down fa-w-14 "
        data-icon="chevron-down"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <button
      className="li-date"
      onClick={[Function]}
    >
      Mar 25th, 2020
    </button>
  </div>
  <ul
    className="dashboard__list"
  />
</div>
`;
